earth_enterprise/src/fusion/autoingest/JsonUtils_unittest.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018 the Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Unit tests for Js Utilities 
+
+#include <string>
+#include <gtest/gtest.h>
+#include "fusion/autoingest/JsonUtils.h"
+
+namespace {
+
+class JsonUtilsAccessor : public JsonUtils {
+  public:
+    std::string RunJsonBool(bool value) {
+      return JsonBool(value);
+    }
+  
+    std::string RunLookAtJson(const std::string& lookAtSpec){
+      return LookAtJson(lookAtSpec);
+    }
+
+    std::string RunJsonObject(const std::map<std::string, std::string>& field_map){
+      return JsonObject(field_map);
+    }
+
+    std::string RunJsonArray(const std::vector<std::string>& array_values){
+      return JsonArray(array_values);
+    }
+};
+
+
+class JsonUnitTest : public testing::Test, public JsonUtils {
+  public:
+    JsonUtilsAccessor jsonUtils;
+};
+
+// Testing the JsonBool function
+TEST_F(JsonUnitTest, JsonBoolTrue) {
+    const bool trueBool = true;
+    const std::string expectedTrue= "true";
+    std::string result = jsonUtils.RunJsonBool(trueBool);

```

Comment:
Minor: I think that since this is a fixture and the test fixture inherits from `JsonUtils`, you can just call `JsonBool` directly. Then you don't need the Accessor class.

Reply:
Or, on the flip side, if you leave it like it is I don't think `JsonUnitTest` needs to inherit from `JsonUtils`.